src/DefaultEtagGenerator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: adri
+ * Date: 06/01/16
+ * Time: 16:04
+ */
+
+namespace ZF\HttpCache;
+
+use Zend\Http\Request as HttpRequest;
+use Zend\Http\Response as HttpResponse;
+
+class DefaultEtagGenerator implements EtagGeneratorInterface
+{
+    /**
+     * Returns an ETag for the given response.
+     *
+     * @param HttpRequest $request
+     * @param HttpResponse $response
+     * @return string Etag
+     */
+    public function generateEtag(HttpRequest $request, HttpResponse $response)
```

Comment:
Since the interface is `ETagGeneratorInterface`, I think we could drop some redundancy here and just call the method `generate()`.